puble.css

```diff
@@ -0,0 +1,117 @@
+.container {
+  width: 564px;
+
+  display: flex;
+  flex-direction: column;
+  border: solid 1px #9a9a9a;
+  color: #9a9a9a;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serifZ;
+}
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 45px 16px 16px 16px;
+  border-bottom: solid 1px #c4c4c4;
+}
```

Comment:
정렬이 맞지 않습니다!
![image](https://user-images.githubusercontent.com/87893624/158068899-64bf4131-ddda-4955-b7a2-28f0268e7ebf.png)

align-items: center; 를 통해 가운데 정렬을 해주세요.
![image](https://user-images.githubusercontent.com/87893624/158068968-8d924e8e-76bb-4e77-8858-db94e9e2ba7c.png)
```suggestion
.header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 45px 16px 16px 16px;
  border-bottom: solid 1px #c4c4c4;
}
```